tests/jballerina-unit-test/src/test/resources/test-src/expressions/binaryoperations/add-operation.bal

```diff
@@ -460,6 +460,62 @@ function testXmlSubtypesAddition() {
     assertEquality(p, result7);
 }
 
+function testXmlAdditionWithQueryExpressions() {
+    xml y = xml `foo` + from string s in ["a", "b"] select xml `${s}`;
+    xml:Text expectedVal = xml `fooab`;
+    assertTrue(y is xml:Text);
```

Comment:
To maintain the consistency, better if this is changed to `y1`.
```suggestion
    xml y1 = xml `foo` + from string s in ["a", "b"] select xml `${s}`;
    xml:Text expectedVal = xml `fooab`;
    assertTrue(y1 is xml:Text);
```

Reply:
will update

Reply:
Updated